dmutils/forms/fields.py

```diff
@@ -46,6 +46,11 @@
 class DMBooleanField(DMFieldMixin, wtforms.fields.BooleanField):
     widget = DMCheckboxInput(hide_question=True)
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if "id" not in kwargs:
+            self.id = self.id + "-1"
```

Comment:
I was a bit confused at first as I thought `DMBooleanField` covered Yes/No radio forms, but it's just a single checkbox, right? Can we add a comment perhaps, to help explain the hardcoded `-1` suffix?

Reply:
Done